package.json

```diff
@@ -76,88 +76,104 @@
     ],
     "commands": [
       {
-        "command": "extension.fabric8AnalyticsWidgetFullStack",
-        "title": "Dependency Analytics Report ..."
+        "command": "extension.stackAnalysis",
```

Comment:
Please fix the prefix as well, I had mentioned in the previous comment.

extension.stackAnalysis => fabric8.stackAnalysis or crda.stackAnalysis

Reply:
ack